src/components/common/TmSessionExplore.vue

```diff
@@ -242,7 +254,7 @@ export default {
     }
   },
   apollo: {
-    addressPrefixes: {
+    networks: {
       query: gql`
         query Network {
           networks {
```

Comment:
Here it should add the experimental parameter. Or if you merge my PR #3669 then just import Networks from `src/gql`

Otherwise you get this:

<img width="380px" src="https://user-images.githubusercontent.com/40721795/76633357-ddb48c00-6544-11ea-8fa6-c0725ae4d06b.png">

Reply:
or merge #3672 as mentioned

Reply:
Oh. I was looking for an Aleksei PR and couldn't find it. There it is

Reply:
but this actually doesn't affect the changes in this PR so can we still merge?

Reply:
The problem is that I cannot test it 😆

Reply:
Coverage is 100% 
![image](https://user-images.githubusercontent.com/5869273/76706178-9bf62380-66b3-11ea-9eaa-7a489a0581e1.png)

Let's merge the other PR first

Reply:
ah true. It is CI what failed. Sorry